tests/TestCases/TestCase.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Tests\TestCases;
+
+use Orchestra\Testbench\TestCase as BaseTestCase;
+
+abstract class TestCase extends BaseTestCase
+{
+    private static $isDBSetup = false;
```

Comment:
I see this as part of the example I sent you earlier, but it doesn't appear to be used anywhere. You seem to be missing the `if (!self::$isDBSetup) { ... }` in the setup function. But also you said its working now? I'm just a little confused

Reply:
Yeah, TestBench seems to handle the setup and teardown of the db on its own, without us needing to guard against multiple instances. I had just missed this line while cleaning it up, removing it now